Resilio Connect API/JSMonitor/app.js

```diff
@@ -36,3 +36,23 @@ setTimeout(function() {console.log(enumerateAgents());}, 5000);
 
 console.log(findArrayDiff([1, 5, 17, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160], [1, 5, 17, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 165]));
 
+var prevList;
+var updatedList = [];
+
+function periodicAgentUpdate() {
```

Comment:
Structure-wise, I would like to separate the calculation of the diff from the action that should be taken when there is a diff.  So the function periodicAgentUpdate() should take a callback function as a parameter and call it when a diff is detected.  Let me know if you need help with this one.